operator/metadata.csv

```diff
@@ -0,0 +1 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
```

Comment:
Since the dashboard uses some `datadog.operator.*` metrics, let's add those metrics here.